src/components/Terminal/XTermDefaultStyle.ts

```diff
@@ -117,12 +117,4 @@ export default `
 	height: 1px;
 	overflow: hidden;
 }
-
-.xterm-dim {
-	opacity: 0.5;
-}
-
-.xterm-underline {
-	text-decoration: underline;
-}
```

Comment:
@thgreasi 	Why are these changed? I didn't remove anything.

Reply:
@dimitrisnl try pulling the latest master, remove node_modules, and see if you get the same result. We removed the package-lock.json, so that might be the culprit.

Reply:
No change - is it safe to include? @sradevski